common/source/docs/common-frsky-yaapu.rst

```diff
@@ -0,0 +1,36 @@
+.. _common-frsky-yaapu:
+
+=============================
+Yaapu FrSky OpenTX LUA Script
```

Comment:
I'm unsure whether to use or not the project name "Yaapu FrSky Telemetry Script" because that's the name that search engines have been indexing for the last 2 years and it would be nice to include the ardupilot wiki among them
If so I would call this "Yaapu FrSky Telemetry LUA Script for OpenTX", dos this make sense?